app/views/simulator/edit.html.erb

```diff
@@ -573,6 +577,8 @@
 <script src="/js/verilog.js"></script>
 <script type="text/javascript" src="/js/metadata.json"></script>
 <script src="/js/canvas2svg.js"></script>
+<script src="/js/data.js"></script>
```

Comment:
why including `/js/data.js` ? 
we are migrating from old UI, so these files will be deleted.

Reply:
Thanx for the review, this file has a **postUserIssue(message)** function declared to register the report , Do you want me to include the same function under script tag in this HTML file?